media-sound/mikmod/mikmod-3.2.8.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="A module player for Unix"
+DESCRIPTION="libmikmod supports many formats, including mod, s3m, it, and xm."
+HOMEPAGE="http://mikmod.sourceforge.net"
+COPYRIGHT="1998-2017 Jean-Paul Mikkers, Jake Stine, Steve McIntyre,
+Frank Loemker, Andrew Zabolotny."
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://sf.net/projects/mikmod/files/mikmod/$portVersion/mikmod-$portVersion.tar.gz"
+CHECKSUM_SHA256="dbb01bc36797ce25ffcab2b3bf625537b85b42534344e1808236ca612fbaa4cc"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	mikmod$secondaryArchSuffix = $portVersion
+	cmd:mikmod$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku${secondaryArchSuffix}
```

Comment:
Ah, probably a remainder of the libmikmod recipe (probably in there too then) ;)